.github/workflows/ci_nightly.yml

```diff
@@ -114,5 +133,9 @@ jobs:
           path: archive/${{ steps.version.outputs.NIGHTLY_VERSION }}/Release-build.log
 
       - name: Send notification
+        if: env.CI_WEBHOOK_URL != '' && env.CI_WEBHOOK_SECRET != ''
+        env:
+          CI_WEBHOOK_URL: ${{ secrets.CI_WEBHOOK_URL }}
+          CI_WEBHOOK_SECRET: ${{ secrets.CI_WEBHOOK_SECRET }}
         run: |
-          /usr/bin/curl -H "X-CI-WebHook: true" -H "Content-Type: application/json" -d '{"secret": "${{ secrets.CI_WEBHOOK_SECRET }}", "repository": "hammerspoon", "workflow": "Dev Build", "message": "New development build: ${{ steps.version.outputs.NIGHTLY_VERSION }}"}' ${{ secrets.CI_WEBHOOK_URL }}
+          /usr/bin/curl -H "X-CI-WebHook: true" -H "Content-Type: application/json" -d '{"secret": "$CI_WEBHOOK_SECRET", "repository": "hammerspoon", "workflow": "Dev Build", "message": "New development build: ${{ steps.version.outputs.NIGHTLY_VERSION }}"}' $CI_WEBHOOK_URL
```

Comment:
Is there a particular reason for pulling the secrets into env here?

Reply:
`If` doesn't have access to secrets. Since the `if` needs access to the existence of the secret and the run needs the secret or seemed vaguely worth just sticking the secret into the env instead of just an existence. But, no, it could just be an existence variable and the run could retain its direct reference to the secret.